message tab_bossShootEffectTable 
{
	optional string tname = 1;
	repeated tab_bossShootEffect tlist = 2;
}

message tab_bossShootEffect
{
	optional int32 id = 1;
	optional string Comment = 2;
	optional int32 InGameName = 3;
	optional int32 LastTime = 4;
}